build/build_and_run_tests.sh

```diff
@@ -32,6 +32,7 @@ run_tests() {
   sbt clean
   sbt -DSPARK_VERSION=$1 test ds-it
 }
+run_tests 2.2.0
```

Comment:
This PR mixes the feature described in its title with upgrade to new spark. Can it be separated?